conversion_proxy/src/lib.rs

```diff
@@ -252,29 +309,43 @@ impl ConversionProxy {
             }
             PromiseResult::Failed => panic!("ERR_FAILED_ORACLE_FETCH"),
         };
+        // Check rate errors
+        assert!(
+            rate.num_error == 0 && rate.num_success == 1,
```

Comment:
Is it possible for `rate.num_success` to be greater than 1? Should it use `rate.num_success >= 1`?